src/html/settings.html

```diff
@@ -4,346 +4,399 @@
+            <label for="enable-sound-signal">
+              <span class="setting">Sound notification at end of interval</span>
+              <span class="description">When the Pomodoro timer interval ends, a sound will be played.</span>
+            </label>
+            <div class="subsettings-details">
+              Set volume
+              <input type="range" id="sound-volume" class="slider">
+              <span id="volume-label">100%</span>
+              <button id="sound-test" class="test">Test</button>
             </div>
-
           </div>
-        </div>
-        <div class="tab" data-tab="integrations">
-          <div class="section">
-            <h2>Integrations</h2>
-            <p>Enable the tools you want to use with Toggl Button. You can enable everything by clicking the "Enable all" button.</p>
-            <div class="permissions-toolbar" data-id="regular">
-              <input type="text" id="permission-filter" placeholder="Filter permissions"/>
-              <a href="javascript:void(0)" id="filter-clear">&times;</a>
-              <div class="button-container">
-                <button class="enable-all">Enable all</button>
-                <button class="disable-all">Disable all</button>
-              </div>
+          <div class="checkbox field pomodoro-ticker">
+            <input type="checkbox" id="enable-ticker-sound">
+            <label for="enable-ticker-sound">
+              <span class="setting">Ticking sound during Pomodoro (beta)</span>
+              <span class="description">While a Pomodoro timer is active, a ticking sound will be played.</span>
+            </label>
+            <div class="subsettings-details">
+              Set volume
+              <input type="range" id="ticker-sound-volume" class="slider">
+              <span id="ticker-volume-label">100%</span>
+              <button id="ticker-sound-test" class="test">Test</button>
             </div>
-            <div id="perm-container"></div>
           </div>
-
-          <div class="section">
-            <h2>Custom Integrations</h2>
-            <p>If you use a tool hosted on a custom domain, you can enable it here. Enter the domain name in the format "toggl.com" and select the integration from the dropdown. <em>Ports are not supported.</em></p>
-            <input type="text" id="new-permission" placeholder="Custom domain url"/>
+        </section>
+      </div>
+      <div class="tab" data-tab="integrations">
+        <section>
+          <div class="headline">
+            <h3>Integrations</h3>
+          </div>
+          <p>Enable the tools you want to use with Toggl Button. You can enable everything by clicking the "Enable all" button.</p>
+          <div class="permissions-toolbar" data-id="regular">
+            <input type="text" id="permission-filter" size="44" placeholder="Filter integrations"/>
+            <a href="javascript:void(0)" id="filter-clear">&times;</a>
```

Comment:
This overflows the parent container and causes a layout jump when you type anything in the integration filter box
![image](https://user-images.githubusercontent.com/1716853/69673240-e67d7100-10bf-11ea-9e77-8473f6b2b948.png)

Also the hit-target is so small, maybe we can postpone cleaning this part for https://github.com/toggl/toggl-button/issues/1572

Reply:
I'll clean this up, didn't realise size would causes these issues :woman_facepalming: